### PR TITLE
New/gum 22 biometric login

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

File: src/screens/login/components/login-form/login-form.tsx [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
+
+import {
+  getBiometricCredentials,
+  hasCredentials as checkBiometry,
+} from 'src/helpers';
+
+import { styles } from './styles';
 import { useAppDispatch } from 'src/hooks';
 import { authActions } from 'src/store/actions';
 import { defaultLoginPayload } from '../../common';
```
Comment (katyakostikova) [inline src/screens/login/components/login-form/login-form.tsx]: wrong import order

File: src/screens/login/login.tsx [hunk 2]
```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { SafeAreaView } from 'react-native-safe-area-context';
 import { View, Image, Text } from 'react-native';
 
-import { LoginForm } from 'src/screens/login/components';
 import { AuthRoute, ButtonMode } from 'src/common/enums';
 import { MainButton } from 'src/components';
+import { LoginForm } from './components';
 import { useAppNavigation } from 'src/hooks';
 import { styles } from './styles';
```
Comment (katyakostikova) [inline src/screens/login/login.tsx]: also import order

File: src/store/auth/actions.ts [hunk 3]
```diff
@@ -2,15 +2,44 @@ import { createAction, createAsyncThunk } from '@reduxjs/toolkit';
...
 import { ISignInPayload } from 'src/common/types';
-import { ActionTypes } from './common';
 import { SecureStorageKey } from 'src/common/enums';
+import { ActionTypes } from './common';
+
+import {
+  hasBiometry,
+  revokeBiometricCredentials,
+  setBiometricCredentials,
+} from 'src/helpers';
```
Comment (katyakostikova) [inline src/store/auth/actions.ts]: also here

File: src/store/auth/actions.ts [hunk 4]
```diff
@@ -2,15 +2,44 @@ import { createAction, createAsyncThunk } from '@reduxjs/toolkit';
...
+
+    await secureStorage.setItem(SecureStorageKey.ACCESS_TOKEN, response.token);
+    dispatch(loadCurrentUser());
+  }
+);
+
+const signInFingerprint = createAsyncThunk(
+  ActionTypes.SIGN_IN_FINGERPRINT,
+  async (
+    { username: email, password }: { username: string; password: string },
```
Comment (katyakostikova) [inline src/store/auth/actions.ts]: maybe you'll create an interface for type?

**Review by katyakostikova — APPROVED** [review]

**Review by yevhenprybytok — APPROVED** [review]

**Pull request merged**